types/react-select/lib/components/MultiValue.d.ts

```diff
@@ -19,36 +20,36 @@ export type MultiValueProps<OptionType> = CommonProps<OptionType> &{
 };
 
 export function multiValueCSS(): React.CSSProperties;
-export function multiValueLabelCSS(props: MultiValueProps<any>): React.CSSProperties;
-export function multiValueRemoveCSS(props: MultiValueProps<any>): React.CSSProperties;
+export function multiValueLabelCSS(props: MultiValueProps<any, any>): React.CSSProperties;
+export function multiValueRemoveCSS(props: MultiValueProps<any, any>): React.CSSProperties;
 
-export interface MultiValueGenericProps<OptionType> {
+export interface MultiValueGenericProps<OptionType, SelectComponentsProps = DefaultSelectComponentsProps> {
   children: ReactNode;
   data: OptionType;
   innerProps: { className?: string };
-  selectProps: any;
+  selectProps: SelectProps<OptionType, SelectComponentsProps>;
 }
 export const MultiValueGeneric: ComponentType<MultiValueGenericProps<any>>;
 
 export const MultiValueContainer: typeof MultiValueGeneric;
 export const MultiValueLabel: typeof MultiValueGeneric;
-export type MultiValueRemoveProps<OptionType> = CommonProps<OptionType> & {
+export type MultiValueRemoveProps<OptionType, SelectComponentsProps = DefaultSelectComponentsProps> = CommonProps<OptionType, SelectComponentsProps> & {
   children: ReactNode,
   innerProps: {
     className: string,
     onTouchEnd: (event: any) => void,
     onClick: (event: any) => void,
     onMouseDown: (event: any) => void,
   },
-  selectProps: any,
```

Comment:
Was this intentionally removed? Or should this have been something like `selectProps: SelectProps<OptionType, SelectComponentsProps>;`?

Reply:
Yes, I've removed it because `selectProps` field is in `CommonProps`.